### ISSUE TITLE
Trivy vulnerability scanner finds outdated library in saml2aws v.2.30.0

### ISSUE THREAD

**Issue by Stanislava27-zz** [Versent/saml2aws#667]
One of my images includes saml2aws v.2.30.0. But [Trivy ](https://github.com/aquasecurity/trivy) finds outdated dependencies after the installation. 
![image](https://user-images.githubusercontent.com/55736625/118949764-d7e0e480-b959-11eb-863a-269da1a5c35a.png)

Do you know when it is going to be updated? 

Thank you :)

**Comment by tony-flatiron** [comment]
There is a new CVE-2021-42836 on the same library. Fixed version is 1.9.3

[mitre](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42836)
[Snyk](https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMTIDWALLGJSON-1766963)

Snyk scan output

```
Testing /Users/anthony.lozano/tmp/codes/saml2aws...

✗ High severity vulnerability found in github.com/tidwall/gjson
  Description: Regular Expression Denial of Service (ReDoS)
  Info: https://snyk.io/vuln/SNYK-GOLANG-GITHUBCOMTIDWALLGJSON-1766963
  Introduced through: github.com/tidwall/gjson@1.8.1
  From: github.com/tidwall/gjson@1.8.1
  Fixed in: 1.9.3

Package manager:   gomodules
Target file:       go.mod
Project name:      github.com/versent/saml2aws/v2
Open source:       no
Project path:      /tmp/codes/saml2aws
Licenses:          enabled

Tested 100 dependencies for known issues, found 1 issue, 1 vulnerable path.`

**Comment by tinaboyce** [comment]
Closing this issue as the current go.mod is using v1.17.0 https://github.com/Versent/saml2aws/blob/f183682460a430642782e958feb8473941a07fbe/go.mod#L25
